packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -193,16 +193,20 @@ function useProcessMedia({
       };
 
       const onUploadStart = () =>
-        updateExistingElements(resourceId, {
+        updateExistingElements(canvasResourceId, {
           trimData,
           isTrimming: true,
         });
 
       const onUploadError = () =>
-        updateExistingElements(resourceId, { isTrimming: false });
+        updateExistingElements(canvasResourceId, { isTrimming: false });
 
       const onUploadSuccess = ({ resource }) => {
-        copyResourceData({ oldResource, resource });
+        const properOldResource = {
```

Comment:
`properOldResource` 🤔

Reply:
Yeah, I know... `oldCanvasResource` is probably a better name, as that's what it is.

Reply:
Yep, `oldCanvasResource` or something similar sounds better 😄

Reply:
Changed in d020e80139.